src/Microsoft.AspNetCore.ApplicationInsights.HostingStartup/ApplicationInsightsStartupLoader.cs

```diff
@@ -22,6 +24,16 @@ public class ApplicationInsightsHostingStartup : IHostingStartup
         public void Configure(IWebHostBuilder builder)
         {
             builder.UseApplicationInsights();
+            builder.ConfigureServices(InitializeServices);
+        }
+
+        /// <summary>
+        /// Adds the Javascript <see cref="TagHelperComponent"/> to the <see cref="IServiceCollection"/>.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> associated with the application.</param>
+        protected virtual void InitializeServices(IServiceCollection services)
```

Comment:
What's the use-case for exposing this guy? It's better to default to less extensible and only add where necessary.